docs/userguide/configuration.rst

```diff
@@ -511,6 +512,41 @@ Default: No time limit.
 Task hard time limit in seconds. The worker processing the task will
 be killed and replaced with a new one when this is exceeded.
 
+.. setting:: task_allow_error_cb_on_chord_header
+
+.. _task_allow_error_cb_on_chord_header:
```

Comment:
We don't need an anchor here. The settings directive creates one for us.

Reply:
Np, will be removed

Reply:
Fix pushed